src/runtime/refiner.ts

```diff
@@ -675,17 +677,110 @@ export class QueryArgumentPrimitive extends RefinementExpression {
   }
 }
 
+export class BuiltIn extends RefinementExpression {
+  evalType: Primitive;
+  value: string;
+
+  constructor(value: string, evalType: Primitive.NUMBER | Primitive.BOOLEAN | Primitive.TEXT) {
+    super('BuiltInNode');
+    this.value = value;
+    this.evalType = evalType;
+  }
+
+  static fromAst(expression: BuiltInNode, _typeData: Dictionary<ExpressionPrimitives>): RefinementExpression {
+    let type = undefined;
+      switch (expression.value) {
+        case 'now()':
+          type = Primitive.NUMBER;
+          break;
+        default:
+          throw new Error(`Unresolved built in name '${expression.value}' in the refinement expression.`);
+      }
+    return new BuiltIn(expression.value, type);
+  }
+
+  toLiteral() {
+    return {
+      kind: this.kind,
+      evalType: this.evalType,
+      value: this.value
+    };
+  }
+
+  static fromLiteral(expr): RefinementExpression {
+    return new BuiltIn(expr.value, expr.evalType);
+  }
+
+  toString(): string {
+    return this.value.toString();
+  }
+
+  toSQLExpression(codeGenerator: CodeGenerator): string {
+    if (this.value === 'now()') {
+      return this.value;
+    }
+    // TODO: Implement SQL getter for 'currentTime'
```

Comment:
what is 'currentTime' - how is it different from now()?

Reply:
I just hadn't had enough coffee when I wrote it. Thank you for catching it.